plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -3010,7 +3010,7 @@ def remove_authz_custom_policy(self, policy_id, client_id, realm):
     def get_authz_permission_by_name(self, name, client_id, realm):
         """Get authorization permission by name"""
         url = URL_AUTHZ_POLICIES.format(url=self.baseurl, client_id=client_id, realm=realm)
-        search_url = "%s/search?name=%s" % (url, name.replace(" ", "%20"))
+        search_url = f"{url}/search?name={name.replace(' ', '%20')}"
```

Comment:
We should use proper quoting functions for this at some point. https://docs.python.org/3/library/urllib.parse.html exists for a reason. (Not in this PR though.)